src/management-interface/src/Endpoints/endpointsStyles.js

```diff
@@ -5,7 +5,7 @@ import { tabsTotalHeight } from '../Tabs/tabStyles'
 
 export const EndpointsContainer = glamorous.section({
   display: 'flex',
-  height: `calc(100vh - ${headerTotalHeight + tabsTotalHeight}px)`,
+  height: `calc(100vh - ${headerTotalHeight + tabsTotalHeight}px - 2rem)`,
```

Comment:
the 2rem seems like an arbitrary value.  How did you come about with it?   We're ultimately taking the total height of the viewport, subtractig the header height, as well as the tabs height.  The result is being padded somehow (maybe there was margin added below the header or tabs or something.  That should be the value we need to equalize from.

Reply:
The 2rem comes from the surrounding container that has a padding of 1rem all around which makes up to 2rem height.

Reply:
https://github.com/pixelprodotco/appstrap/blob/master/src/management-interface/src/App.js#L20

Reply:
nice - ok good.